app/uk/gov/hmrc/hmrcemailrenderer/templates/hts/htsEmailVerification.scala.html

```diff
@@ -1,10 +1,11 @@
 @(params: Map[String, Any])
 @uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Verify your email address") {
-<p style="margin: 0 0 30px; font-size: 19px;">Dear @{params("name")}</p>
-<p style="margin: 0 0 30px; font-size: 19px;">You received this email because you chose to provide a new email address for Help to Save.</p>
-<p style="margin: 0 0 30px; font-size: 19px;">You'll need to verify your email address within the next 30 minutes.</p>
-<p style="margin: 0 0 30px; font-size: 19px;">To verify your email address click the link below:</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Hello @{params("name")}</p>
```

Comment:
Is there a reason you changed your salutation from "Dear" to "Hello"? Dear would be preferred

Reply:
Atm all other Help to Save comms to users address them with Hello, so we are using this for consistency

Reply:
Ok we are going to change it to Dear